Xamarin.Forms.Platform.iOS/ContextActionCell.cs

```diff
@@ -87,7 +87,7 @@ public override void LayoutSubviews()
 		{
 			base.LayoutSubviews();
 
-			if (_scroller == null || (_scroller != null && _scroller.Frame.Width == ContentView.Bounds.Width))
```

Comment:
When "_scroller.Frame.Width == ContentView.Bounds.Width" is true then no further layout is happening and cell view is broken